api/envoy/service/status/v2/csds.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+package envoy.service.status.v2;
```

Comment:
Should we start this as a v2alpha/v3alpha API or do you want to go straight to stable?

Reply:
My take is we go straight to stable, this is a pretty straightforward API surface.

Reply:
OK sounds good.